### PR TITLE
GPII-2287: Moved build of test-window.c into Tests.ps1

### PR DESCRIPTION
**Pull request by javihernandez**
This pr is meant to land hst-2017, not master (yet). @gloob just confirm that this is what you were asking for. @kaspermarkus once we have an ok from @gloob, can you review/merge this?

### CODE REVIEW

**Comment by gloob** [comment]
It looks like what we wanted.

It's missing the deletion of that part in the Build.ps1 script and the inclusion (of Tests.ps1 invocation) in package.json or explicitely in the Vagrantfile.

**Comment by javihernandez** [comment]
@gloob right, I've just updated it.

**Comment by gloob** [comment]
> ==> default: VERBOSE: 'C:\Windows\Microsoft.NET\Framework64\v4.0.30319\msbuild.exe' exited with 0
> ==> default: ./provisioning/Tests.ps1 : The term './provisioning/Tests.ps1' is not recognized as the name of a cmdlet, function, 
> ==> default: 
> ==> default: script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is 
> ==> default: correct and try again.
> ==> default: At line:1 char:1
> ==> default: + ./provisioning/Tests.ps1
> ==> default: + ~~~~~~~~~~~~~~~~~~~~~~~~
> ==> default:     + CategoryInfo          : ObjectNotFound: (./provisioning/Tests.ps1:String) [], CommandNotFoundException
> ==> default:     + FullyQualifiedErrorId : CommandNotFoundException
> ==> default:  

Interestingly the CI system silent these error. This is a bug.

File: provisioning/Build.ps1 [hunk 1]
```diff
@@ -40,7 +40,4 @@ $msbuild = Get-MSBuild "4.0"
 $listenersDir = Join-Path $mainDir "listeners"
 Invoke-Command $msbuild "listeners.sln /nodeReuse:false /p:Configuration=Release /p:FrameworkPathOverride=`"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5.1`"" $listenersDir
 
-Invoke-Environment "C:\Program Files (x86)\Microsoft Visual C++ Build Tools\vcbuildtools.bat"
-$testProcessHandlingDir = Join-Path $mainDir "gpii\node_modules\processHandling\test"
-Invoke-Command "cl" "test-window.c" $testProcessHandlingDir
-rm (Join-Path $testProcessHandlingDir "test-window.obj")
+Invoke-Expression "./provisioning/Tests.ps1"
```
Comment (gloob) [inline provisioning/Build.ps1]: This should be something like:

`Invoke-Expression (Join-Path $scriptDir "Tests.ps1")`

**Comment by gloob** [comment]
Nice, it looks good to me.

**Comment by javihernandez** [comment]
ok to test

**Pull request merged**